.github/actions/bootstrap/action.yml

```diff
@@ -56,3 +61,14 @@ runs:
         # If source files changed but packages didn't, rebuild from a prior cache.
         restore-keys: |
           ${{ runner.os }}-nextjs-${{ hashFiles('**/package-lock.json') || hashFiles('**/yarn.lock') }}-
+    - name: Cache node modules for others
+      uses: actions/cache@v3
+      if: ${{ inputs.framework != 'next' }}
+      with:
+        path: |
```

Comment:
Is this necessary for oneliner? Is this preparation for another command?

Reply:
Probably not 😅